tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -935,6 +950,14 @@ trait LevelNames {
     }
 }
 
+#[cfg(feature = "ansi")]
+impl LevelNames for Pretty {
+    const TRACE_STR: &'static str = "TRACE";
+    const DEBUG_STR: &'static str = "DEBUG";
+    const INFO_STR: &'static str = " INFO";
+    const WARN_STR: &'static str = " WARN";
+    const ERROR_STR: &'static str = "ERROR";
+}
 impl LevelNames for Full {
```

Comment:
very silly formatting nit: can we add a whitespace here:
```suggestion
}

impl LevelNames for Full {
```

Reply:
Fixed in c017573 :sparkles: